wiki/People/The_Team/Nomination_Assessment_Team/en.md

```diff
@@ -9,35 +9,63 @@ tags:
 
-Before joining the NAT, a user must either be a full member of the Beatmap Nominators, or previously an NAT member and still involved in the community. Most NAT candidates are initially considered based on their commitment to helping the mapping and modding community, and further demonstration of their ability to contribute to all four subcategories of NAT responsibilities is usually the basis for being promoted.
+Before joining the NAT, a user must either be a full member of the Beatmap Nominators, or previously an NAT member and still involved in the community. Most NAT candidates are initially considered based on their commitment to helping the mapping and modding community, and further demonstration of their ability to contribute to a multitude of NAT responsibilities is usually the basis for being promoted.
 
-Since the majority of new NAT members will help with BN evaluation, it's important that NAT candidates are very proficient in assessing the proficiency of others. Full BNs have the option to do mock evaluations alongside the NAT's real ones, giving them an opportunity to practice. NAT candidates have a much better chance of being promoted if their evaluations are thorough and come to similar conclusions as the NAT (or have supportive reasoning otherwise).
+Since all new NAT members start off as evaluators, it's important that NAT candidates are exceptional in assessing the proficiency of others. Full BNs may be given the opportunity to participate in mock evaluations alongside the NAT's real ones, giving them an opportunity to practice. NAT candidates have a much better chance of being promoted if their evaluations are thorough and come to similar conclusions as the NAT (or have supportive reasoning otherwise).
 
 The NAT keep tabs on potential NAT candidates over long periods of time, and occasionally convene to decide if a candidate should be promoted, similar to how BN evaluations are done. BNs are also allowed to ask about joining the NAT if they want to receive feedback and make sure they're being considered. However, depending on the activity and skillset of the current NAT, there may not be a need for new members. New NAT will likely be promoted only when one of the current members is becoming less active, or there are new workloads requiring more members, etc.
 
+###Becoming an advisor
+
+Evaluators have the chance to become advisors if they've shown themselves capable of properly handling the advisory tasks outlined above. As long as there are enough evaluators to keep up with the required workload, an evaluator may announce their decision to become an advisor to the rest of the NAT, explicitly stating either their existing contributions or their planned contributions. They will then be subject to activity expectations depending on their specific situation. An evaluator may be deemed unsuitable for an advisory position if their proposed or existing contributions are seen as lackluster or unnecessary. 
```

Comment:
more in-line with what we discussed

the nat member just announcing to the team that they're suddenly an advisor is very different from having you gauge and discuss this with them before anything else

important we get this in writing so we don't forget in like a years time who the nat should ask about this stuff

```suggestion
### Becoming an advisor

Evaluators that show proficiency in the advisory tasks outlined above, to the point where their current or planned contribution is indispensable and far outweighs their necessity as an evaluator, can become advisors. This is as judged by a designated member in the team capable of this, currently [UberFazz](https://osu.ppy.sh/users/8646059), whom NAT members may ask advisor-related questions to. The new advisor will then be subject to activity expectations depending on their specific situation. If the contribution of an advisor is considered lackluster or unnecessary at some later point, again as judged by the designated member, then they will be moved back to an evaluator.
```

Reply:
nitpick but could add ::{ flag=US }:: next to UberFazz in Naxess' suggestion for wiki consistency's sake